homoalg.tex

```diff
@@ -1332,10 +1332,10 @@ \section{Garben}
 % I.5.5 (Keime und Halme)
 
 \begin{defn}
-  Sei $\Fais$ eine Garbe auf $X$. Der \emph{Halm} von $\Fais$ in $x \in X$ ist
+  Sei $\Fais$ eine Garbe auf $X$. Der \emph{Halm} o. die \emph{Faser} von $\Fais$ in $x \in X$ ist
```

Comment:
Ah okay. Der Hinweis wär dann nicht schlecht ja :D 

Was sind denn eigtl die englischen Begriffe? Irgendwie dachte ich halt halm = fibre = faser.

Wenn du magst könnte evtl noch ein kurzer Hinweis in die Zusammenfassung, was du mit Fasern der Strukturgarbe meinst (in der Def von lokal geringten Räumen ).